src/Entity/TimestampableTrait.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/**
+ * @see       https://github.com/zendframework/zend-expressive-authentication-oauth2 for the canonical source repository
+ * @copyright Copyright (c) 2017 Zend Technologies USA Inc. (https://www.zend.com)
+ * @license   https://github.com/zendframework/zend-expressive-authentication-oauth2/blob/master/LICENSE.md
+ *     New BSD License
+ */
+
+declare(strict_types=1);
+
+namespace Zend\Expressive\Authentication\OAuth2\Entity;
+
+use DateTime;
+use DateTimeZone;
+
+trait TimestampableTrait
+{
+    protected $createdAt;
+
+    protected $updatedAt;
```

Comment:
Sorry, maybe I wasn't clear in my previous comment, but it's good to keep PHPDocs here (on class properties) because we don't know what is the type - in methods we know if from method signature.